README.md

```diff
@@ -1,15 +1,15 @@
-# Zaius Node SDK
+# ODP Node SDK
 
-A lightweight Node SDK for sending events and data to Zaius from a Node JS app.
+A lightweight Node SDK for sending events and data to ODP from a Node JS app.
 
 > <b>⚠️ WARNING: This is NOT a browser compatible SDK.</b><br />
-> To interface with Zaius from a web site, use our [Web SDK](https://docs.developers.zaius.com/web-sdk/)
+> To interface with ODP from a web site, use our [Web SDK](https://docs.developers.zaius.com/web-sdk/)
 
 ## Documentation
 
-See the generated [Technical Documentation](https://node-sdk.docs.developers.zaius.com/latest/) for details on every method.
+See the generated [Technical Documentation](https://optimizely-parent.readme.io/optimizely-connect-platform/docs/node-sdk-api-reference-overview) for details on every method.
```

Comment:
Is the `optimizely-parent` subdomain final or are we planning to change that?

Reply:
great catch! thanks! fixed